src/requests/features.jl

```diff
@@ -157,7 +148,7 @@ end
 
 JSONRPC.parse_params(::Type{Val{Symbol("textDocument/documentLink")}}, params) = DocumentLinkParams(params) 
 function process(r::JSONRPC.Request{Val{Symbol("textDocument/documentLink")},DocumentLinkParams}, server)
-    if !haskey(server.documents, URI2(r.params.textDocument.uri))
+    if !hasdocument(server, URI2(r.params.textDocument.uri))
         error("Received 'textDocument/documentLink for non-existing document.")
     end
     links = Tuple{String,UnitRange{Int}}[]
```

Comment:
Note: not currently supported in InitOptions so comment out/remove

Reply:
https://github.com/julia-vscode/LanguageServer.jl/pull/542